src/common/get-key-content-or-null.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
```

Comment:
Maybe we use cosmiconfig to search for `prettierIgnore`?

```js
cosmiconfig("prettierIgnore", {searchPlaces: ["package.json"]})
```

Reply:
My biggest gripe about this is that cosmiconfig is suited for JSON and YAML configuration files, not lists of file globs with or without negations (which ignore files are).

I will try it, however.

Reply:
I've changed the implementation - performance is almost the same compared to the previous method while it's a bit nicer on the eyes (I think).